### DISCUSSION TITLE
backlight scrolling should have an interval option

### DISCUSSION THREAD

**Discussion by RewardedIvan** [polybar/polybar#2690] [Ideas]
If the pulseaudio module has an interval, then why not? Also this could be fixed when adding a [unified module click handler](https://github.com/polybar/polybar/issues/1524).
Do you guys think the pulseaudio module @ 0% volume should be considered muted? Because there isn't a volume specific formatting.

**Comment by patrick96** [comment]
> If the pulseaudio module has an interval, then why not?

Mainly because no one added it ;)
I'll create a feature request issue for this.

> Also this could be fixed when adding a https://github.com/polybar/polybar/issues/1524.

That's not really related.

>  Do you guys think the pulseaudio module @ 0% volume should be considered muted? Because there isn't a volume specific formatting.

No. "muted" is a separate state reported by pulse and is largely independent from volume (you can have for example have a muted sink with 50% volume).
